README.md

```diff
@@ -1,15 +1,111 @@
-# Group_G_UNO-Card-Game
-Git Repository for SYSC5709F Project
+# Group G - UNO Express Card Game
 
-Group Information
-Organization: Carleton Unviersity (SYSC 5709F)
+* [Application Overview](#application-overview)
+* [Group Information](#group-information)
+* [File Structure](#file-structure)
+* [How to Compile and Test](#how-to-compile-and-test)
+* [Documentation](#documentation)
 
+## Application Overview
 
-***********************************************************
-Authors:
-1. Mia Wei (GitHub: MiaYWei)
-2. Krithiga Subramaniam Ravichandran (GitHub: krithigasr)
-3. Hongbo Pang (GitHub: HongBo713)
-4. Aditi Bhardwaj (GitHub: aditibhardwaj98)
+UNO Express is a fun and interesting game, traditionally played by a group of 2 to 4 people with a  deck of printed cards. We are proposing to develop this interactive game in C programming  language which supports a single player competing against a computer opponent. Playing games online  with computers could speed up our responses and develop logic skills. 
+
```

Comment:
Seems like there are some double character space between two words. Let's check it again, and remove them.

Reply:
Made the changes, it shouldn't be a problem when it's rendered anyway.